community/values/src/main/java/org/neo4j/values/storable/PointValue.java

```diff
@@ -280,9 +280,32 @@ public static PointValue fromMap( MapValue map )
     private static Pattern quotesPattern = Pattern.compile( "^[\"']|[\"']$" );
 
     public static PointValue parse( CharSequence text )
+    {
+        return PointValue.parse( text, null );
+    }
+
+    public static PointValue parse( CharSequence text, AnyValue[] fieldsFromHeader )
```

Comment:
How about renaming `fieldsFromHeader` to something like `defaultFields` to make it user agnostic?